Dispatcher/Views/ArticlesTableCells/ArticleCell.swift

```diff
@@ -24,39 +24,62 @@ class ArticleCell: UITableViewCell {
     @IBOutlet weak var tagLabel: UILabel!
     @IBOutlet weak var autherLabel: UILabel!
     @IBOutlet weak var subTitleLabel: UILabel!
-    @IBOutlet weak var dispatchButton: UIButton!
     @IBOutlet weak var moreTagsLabel: UILabel!
     @IBOutlet weak var favoritesButton: UIButton!
+    @IBOutlet weak var articleImage: UIImageView!
     
+    var dispatchButton: DispatcherAppButton?
     var delegate: ArticleCellDelegate?
     var id: String = ""
     
-    
     override func awakeFromNib() {
         super.awakeFromNib()
         
         setCellMarginsAndBorder()
         setTextColorForCellElements()
         setCornerRadiusForCellElements()
-        
         favoritesButton.configuration?.contentInsets = NSDirectionalEdgeInsets(top: 5, leading: 5, bottom: 5, trailing: 5)
+        setupDispatchButton()
+    }
+    
+    private func setupDispatchButton() {
+        let frame = CGRect(x: 18, y: 397, width: 290, height: 36)
+        dispatchButton = DispatcherAppButton(
+            frame: frame,
+            type: .primary,
+            title: AppConstants.navigateToDispatchText,
+            icon: UIImage(named: "Arrow - Right"),
+            iconPosition: .end
+        )
+
+        dispatchButton!.addTarget(self, action: #selector(navigateToDisptachPressed), for: .touchUpInside)
```

Comment:
Usually we want to be informed about the press action on the button through a delegate and not by adding a touchUp selector to the button from the screen in which the button is implemented.